### PR TITLE
update some microsite links & scaffolder to beta lifecycle

### PR DESCRIPTION
**Pull request by benjdlambert**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by timbonicus — APPROVED** [review]
:shipit:

File: plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPage.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const ScaffolderPageContents = () => {
         pageTitleOverride="Create a New Component"
         title={
           <>
-            Create a New Component <Lifecycle alpha shorthand />
+            Create a New Component <Lifecycle shorthand />
```
Comment (freben) [inline plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPage.tsx:60]: No lifecycle at all? Does this default to beta if you leave it out? Das cray

Comment (timbonicus) [reply plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPage.tsx:60]: Presumably you're not using `<Lifecycle />` at all if it's in production 😛

Comment (freben) [reply plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPage.tsx:60]: Sure but, a great api it does not make :)

Comment (benjdlambert) [reply plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPage.tsx:60]: No I noticed that but didn't want to change all that in one PR

**Pull request merged**